sections/infrastructure.include

```diff
@@ -3836,95 +3834,93 @@
+    <dt><var>element</var> = <var>collection</var> . {{HTMLAllCollection/item()|item}}(<var>name</var>)</dt>
+    <dt><var>collection</var> = <var>collection</var> . {{HTMLAllCollection/item()|item}}(<var>name</var>)</dt>
+    <dt><var>element</var> = <var>collection</var> . {{HTMLAllCollection/namedItem()|namedItem}}(<var>name</var>)</dt>
+    <dt><var>collection</var> = <var>collection</var> . {{HTMLAllCollection/namedItem()|namedItem}}(<var>name</var>)</dt>
     <dt><var>element</var> = <var>collection</var>(<var>name</var>)</dt>
     <dt><var>collection</var> = <var>collection</var>(<var>name</var>)</dt>
     <dt><var>element</var> = <var>collection</var>[<var>name</var>]</dt>
     <dt><var>collection</var> = <var>collection</var>[<var>name</var>]</dt>
-    <dd>
-      Returns the item with <a>ID</a> or name <var>name</var> from the collection.
+    <dd>Returns the item with <a>ID</a> or name <var>name</var> from the collection.
 
-      If there are multiple matching items, then an {{HTMLCollection}} object containing
-      all those elements is returned.
+    If there are multiple matching items, then an {{HTMLCollection}} object containing all those
+    elements is returned.
 
-      Only <{button}>, <{form}>, <{iframe}>, <{input}>,
-      <{map}>, <{meta}>, <{object}>, <{select}>, and
-      <{textarea}> elements can have a name for the purpose of this method; their name is
-      given by the value of their <code>name</code> attribute.
-    </dd>
+    The <{formelements/name}> attribute's value provides a name for <{button}>, <{input}>, 
+    <{select}>, and <{textarea}>. Similarly, <{iframe}>'s <{iframe/name}>, <{object}>'s 
+    <{object/name}>, <{meta}>'s <{meta/name}>, <{map}>'s <{map/name}>, and <{form}>'s <{form/name}> 
+    attribute's value provides a name for their respective elements. Only the elements mentioned 
+    have a name for the purpose of this method.</dd>
   </dl>
 
   The object's <a>supported property indices</a> are as defined for <code>HTMLCollection</code>
   objects.
 
-  The following elements are considered <dfn id="all-named-elements">"all"-named elements</dfn>:
-  <{a}>, <{applet}>, <{button}>, <{embed}>, <{form}>, <{frame}>, <{frameset}>, <{iframe}>,
-  <{img}>, <{input}>, <{map}>, <{meta}>, <{object}>, <{select}>, and <{textarea}>
-
   The <a spec="webidl">supported property names</a> consist of the non-empty values of all the
-  <code>id</code> attributes of all the elements <a>represented by the collection</a>, and the
-  non-empty values of all the <code>name</code> attributes of all the <a>"all"-named element or
-  elements</a> <a>represented by the collection</a>, in <a>tree order</a>, ignoring later
-  duplicates, with the <code>id</code> of an element preceding its <code>name</code> if it contributes both, they
-  differ from each other, and neither is the duplicate of an earlier entry.
+  <{global/id}> and <code>name</code> attributes of all the elements
```

Comment:
OK.

Note: this is more widespread in this section (all collections of various flavors use <code>name</code>)